server/data_processors/DashboardDataProcessor/DashboardDataProcessor.test.js

```diff
@@ -49,43 +50,41 @@ describe('Data Processors - Dashboard', () => {
         category: 'reading',
       }),
     ]
-    const db = createDb()
     const jumpVariableData = [
       createSubjectDayData({
+        collection: 'jump-of-collection',
```

Comment:
this property shouldn't exist in the subjects data

Reply:
e42ee3084c5d27c1c4ba2f3b3590560d8f5fa3bf